mtcpclient/closure.go

```diff
@@ -2,14 +2,16 @@ package mtcpclient
 
 import (
 	"fmt"
-	"github.com/dachad/tcpgoon/debugging"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
+
+	"github.com/dachad/tcpgoon/debugging"
 )
 
-func StartBackgroundClosureTrigger(gc GroupOfConnections) <-chan bool {
+// StartBackgroundClosureTrigger creates proper channels to know when to close execution
```

Comment:
thanks for the documentation! But it also triggers a goroutine that monitors if the closure conditions are met (starting then the closure of the program)

Reply:
👍